.github/workflows/acceptance-tests.yml

```diff
@@ -53,6 +53,7 @@ jobs:
     - uses: actions/checkout@v4
     - uses: dorny/paths-filter@v2
       id: filter
+      if: ${{ inputs.test_group == '' && env.mustTrigger == 'false' }}
```

Comment:
just run file change detector if:
 inputs.test_group == '' -> if a test group is selected then we want to run only those tests.
env.mustTrigger == 'false'   -> if mustTrigger=true we don't need to run file changes as we know that all tests will be run